examples/external-lib-example/requirements.txt

```diff
@@ -0,0 +1,3 @@
+spin-sdk == 2.0.0
+componentize-py == 0.11.2
+http-router
```

Comment:
I think we should add a version number here in case there's a new, breaking release in the future.